src/codegen/llvm.zig

```diff
@@ -1967,7 +1970,6 @@ pub const Object = struct {
                 if (!ty.hasRuntimeBitsIgnoreComptime(mod)) {
                     const debug_enum_type = try o.makeEmptyNamespaceDebugType(owner_decl_index);
                     try o.debug_type_map.put(gpa, ty, debug_enum_type);
-                    try o.debug_enums.append(gpa, debug_enum_type);
```

Comment:
Did you mean to remove this line?

Reply:
Yes, llvm [errors](https://github.com/llvm/llvm-project/blob/6009708b4367171ccdbf4b5905cb6a803753fe18/llvm/lib/IR/Verifier.cpp#L1356-L1363) with `invalid enum type` if it contains anything other than `!DICompositeType(tag: DW_TAG_enumeration_type, ...)`.